docs/contributors.yaml

```diff
@@ -402,13 +402,14 @@
   jira_user: yanyanhu
   reviewboard_user: yanyanhu
 
-- name: Yong Qiao Wang
```

Comment:
If your name and reviewboard username have also changed along with your affiliation, then it's probably worth to have two different entries in the YAML file.

```
- name: Yong Qiao Wang
  affiliations:
    - {organization: IBM}
  emails:
    - yqwyq@cn.ibm.com
  jira_user: gradywang
  reviewboard_user: JamesYongQiaoWang

- name: Yongqiao Wang
  affiliations:
    - {organization: Huawei}
  emails:
    - grady.wang@foxmail.com
  jira_user: gradywang
  reviewboard_user: gradywang
```